src/autocomplete.js

```diff
@@ -0,0 +1,8 @@
+// PLACES
+export let placesAutocomplete = places({
```

Comment:
Since this is not changing, use `const` instead.
```suggestion
export const placesAutocomplete = places({
```